packages/algoliasearch-helper/index.d.ts

```diff
@@ -9,6 +9,11 @@ import type {
   SearchResponse,
   SnippetResult,
 } from './types/algoliasearch';
+import type {
+  RecommendationsQuery,
+  RecommendedForYouQuery,
+  TrendingQuery,
+} from '@algolia/recommend';
```

Comment:
won't this break if you don't have recommend in the dependency tree? without ts-ignore I think it may cause an error

Reply:
Oh good call, this could be problematic. In most cases this package would exist through `algoliasearch`, but I've adedd a ts-ignore comment for this import.

Reply:
It won't exist for existing users that just update InstantSearch but not the client (many people probably).

Also I believe there's some tricky stuff with minification etc that makes the ts-ignore not apply to multiple lines in some cases. Maybe not the case here as this is a hand-written .d.ts, but worth checking in a plain project and comparing with the algoliasearch.d.ts file :)

Reply:
I checked and the build script doesn't interact with type files, they are untouched in the package.